components/autoComplete/autoComplete.js

```diff
@@ -234,7 +234,7 @@ class AutoComplete extends Component {
     if (!this.state.inputValue) {
       return str;
     }
-    const value = this.state.inputValue;
+    const value = this.state.inputValue.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
```

Comment:
this input string is basic for compose the group capturing... in case when this string includes a brackets, we'll got the SyntaxError Exception... to solve it, all special symbol were escaping... for example: input string = 'Pekin (' => SyntaxError in RegEx pattern as /(Pekin ()/